src/RobotLoader/RobotLoader.php

```diff
@@ -116,6 +127,18 @@ public function addDirectory($path)
 
 
 	/**
+	 * Excludes path or paths from list.
+	 * @param  string|string[]  absolute path
+	 * @return static
+	 */
+	public function excludeDirectory($path)
```

Comment:
Yes, it could be. But if so, it should be done also in `RobotLoader::addDirectory()`. The functions are almost the same.